### PR TITLE
Remove displayable filter on posts

### PR DESCRIPTION
**Pull request by jarrel-b**
Removing the displayable criteria for posts so that tokens can still be viewed.

### CODE REVIEW

File: graphql/dataloader/dataloaders.helpers.go [hunk 1]
```diff
@@ -90,99 +90,33 @@ func withMaxBatchOne(batchSize int) func(*settings) {
...
 }
 
-func withWaitTime(t time.Duration) func(*settings) {
+func withMaxWait(t time.Duration) func(*settings) {
 	return func(s *settings) {
 		s.waitTime = t
 	}
 }
 
-func withContext(ctx context.Context) func(interface {
```
Comment (jarrel-b) [inline graphql/dataloader/dataloaders.helpers.go:99]: Didn't realize these helpers were already here! I updated them to update settings instead.

**Review by Robinnnnn — APPROVED** [review]

File: graphql/dataloader/dataloaders.go [hunk 2]
```diff
@@ -238,6 +239,12 @@ func NewLoaders(ctx context.Context, q *db.Queries, disableCaching bool) *Loader
 		AutoCacheWithKey: func(token db.Token) persist.DBID { return token.ID },
 	})
 
+	loaders.TokenByTokenIDIgnoreDisplayable = NewTokenLoaderByID(
+		defaultSettingsPlusOpts(ctx, disableCaching, &subscriptionRegistry, &mutexRegistry, withPublishResults(false)),
```
Comment (radazen) [inline graphql/dataloader/dataloaders.go:243]: Good call on not publishing the results! I was initially thinking that if we made `displayable` a parameter in the regular `TokenByTokenID` fetcher, we wouldn't need to do this...but every other query that fetches tokens would need to qualify it with a `displayable` parameter, too. This seems cleaner.

File: graphql/dataloader/dataloaders.helpers.go [hunk 3]
```diff
@@ -90,99 +90,33 @@ func withMaxBatchOne(batchSize int) func(*settings) {
...
-}) {
-	return func(s interface {
-		setContext(ctx context.Context)
-		setWait(time.Duration)
-		setMaxBatch(int)
-		setDisableCaching(bool)
-		setPublishResults(bool)
-	}) {
-		s.setWait(wait)
+func withContext(ctx context.Context) func(*settings) {
```
Comment (radazen) [inline graphql/dataloader/dataloaders.helpers.go:99]: Man, I spent a _really long_ time trying to make these as concise as possible. Cannot believe I missed this obvious syntax.

**Review by radazen — APPROVED** [review]
🔥 🔥 🔥

**Pull request merged**